src/components/form/Form.tsx

```diff
@@ -0,0 +1,160 @@
+            label='Nickname'
+            placeholder='Nickname'
+            {...form.getInputProps('nickname')}
+          />
+          <TextInput
+            className='each'
+            label='Name'
+            placeholder='Name'
+            {...form.getInputProps('name')}
+          />
+          <TextInput
+            className='each'
+            label='Surname'
+            placeholder='Surname'
+            {...form.getInputProps('surname')}
+          />
+          <Select
+            className='each'
+            label='Sex'
+            placeholder='Not selected'
+            {...form.getInputProps('sex')}
+            data={['man', 'woman']}
+          />
+        </Stepper.Step>
+
+        <Stepper.Step label='2'>
+          <label>Advantage</label>
+          {form.values.advantages.map((advantage, index) => (
+            <Group key={index} className='advantage'>
+              <TextInput
+                key={index}
+                mt='md'
+                placeholder='Placeholder'
+                {...form.getInputProps(`advantages.${index}`)}
+                style={{ flex: 1 }}
+              />
+              <MdDelete size={33} fill='#959ead' onClick={() => removeString(index)} />
+            </Group>
+          ))}
+          <Button className='btnAdd' type='button' onClick={addString}>
+            +
+          </Button>
+          <Checkbox.Group label='Checkbox group' {...form.getInputProps('checkbox')}>
+            <Group mt='xs'>
+              <Checkbox value='1' label='1' />
+              <Checkbox value='2' label='2' />
+              <Checkbox value='3' label='3' />
+            </Group>
+          </Checkbox.Group>
+
+          <Radio.Group label='Radio group' {...form.getInputProps('radio')}>
+            <Group mt='xs'>
+              <Radio value='1' label='1' />
+              <Radio value='2' label='2' />
+              <Radio value='3' label='3' />
+            </Group>
+          </Radio.Group>
+        </Stepper.Step>
+
+        <Stepper.Step label='3'>
+          <Textarea
+            {...form.getInputProps('about')}
+            label='About'
+            placeholder='Placeholder'
+            autosize
+            maxRows={200}
+          />
+        </Stepper.Step>
+      </Stepper>
+
+      <Group justify='flex-end' mt='xl'>
+        <div className='btns'>
+          {active === 0 ? (
+            <Link to='/'>
+              <Button variant='default' onClick={prevStep}>
+                Back
+              </Button>
+            </Link>
+          ) : (
+            <Button variant='default' onClick={prevStep}>
+              Back
+            </Button>
+          )}
+          {active !== 2 ? (
+            <Button className='btnNext' onClick={nextStep}>
+              Next step
+            </Button>
+          ) : (
+            <Button className='btnNext' onClick={sendDataApi} disabled={loading}>
+              {loading ? 'Sending...' : 'Send'}
+            </Button>
```

Comment:
use loader for Button from [mantine](https://mantine.dev/core/button/#loader-props)